### PR TITLE
Paper - small editorial corrections

### PR DESCRIPTION
**Pull request by Matthew-Jennings**
See https://github.com/openjournals/joss-reviews/issues/4555#issuecomment-1247632276

### CODE REVIEW

**Review by sjswerdloff — APPROVED** [review]

**Comment by Matthew-Jennings** [comment]
Hi @SimonBiggs, @sjswerdloff & @osorenson,

I'm a little at a loss as to why "Mone Carlo" and "Winston Lutz" are not capitalised in the built PDF, but are in [`paper.bib`](https://github.com/pymedphys/pymedphys/blob/94724d75a2f8f55014171350e54bd0ff410b9542/paper.bib#L111). Do either of you happen to know the likely cause offhand?

**Comment by sjswerdloff** [comment]
> Hi @SimonBiggs, @sjswerdloff & @osorensen,
> 
> I'm a little at a loss as to why "Monte Carlo" and "Winston-Lutz" are not capitalised in the built PDF, but are in [`paper.bib`](https://github.com/pymedphys/pymedphys/blob/94724d75a2f8f55014171350e54bd0ff410b9542/paper.bib#L111). Do either of you happen to know the likely cause offhand?

No Clue.  Can you provide a link to the PDF (I have gotten to it in the past, but I'm having a rough time finding it right now).

**Comment by Matthew-Jennings** [comment]
> > Hi @SimonBiggs, @sjswerdloff & @osorensen,
> > I'm a little at a loss as to why "Monte Carlo" and "Winston-Lutz" are not capitalised in the built PDF, but are in [`paper.bib`](https://github.com/pymedphys/pymedphys/blob/94724d75a2f8f55014171350e54bd0ff410b9542/paper.bib#L111). Do either of you happen to know the likely cause offhand?
> 
> No Clue. Can you provide a link to the PDF (I have gotten to it in the past, but I'm having a rough time finding it right now).

No worries: https://github.com/pymedphys/pymedphys/suites/8301605791/artifacts/364224927

**Comment by sjswerdloff** [comment]
Acquired clue:
https://tex.stackexchange.com/questions/10772/bibtex-loses-capitals-when-creating-bbl-file
which contains an answer that says:

```
This is not a bug; it's a feature. Depending on the bibliography style in use, BibTeX converts all characters in the title to lowercase, with the exception of the first character. If you want to override this, wrap the character(s) in curly braces, e.g.,

title = "Pascal, {C}, {Java}: were they all conceived in {ETH}?"
```

So I think you want to convert Monte Carlo to {Monte} {Carlo} in the .bib file

**Comment by Matthew-Jennings** [comment]
> Acquired clue: https://tex.stackexchange.com/questions/10772/bibtex-loses-capitals-when-creating-bbl-file which contains an answer that says:
> 
> ```
> This is not a bug; it's a feature. Depending on the bibliography style in use, BibTeX converts all characters in the title to lowercase, with the exception of the first character. If you want to override this, wrap the character(s) in curly braces, e.g.,
> 
> title = "Pascal, {C}, {Java}: were they all conceived in {ETH}?"
> ```
> 
> So I think you want to convert Monte Carlo to {Monte} {Carlo} in the .bib file
> 
> If you have run out of time and want me to take a crack at it, I will try

Thanks! Sorry I missed this! If you haven't started yet, I should be able to give it a crack.

**Comment by osorensen** [comment]
> Acquired clue: https://tex.stackexchange.com/questions/10772/bibtex-loses-capitals-when-creating-bbl-file which contains an answer that says:
> 
> ```
> This is not a bug; it's a feature. Depending on the bibliography style in use, BibTeX converts all characters in the title to lowercase, with the exception of the first character. If you want to override this, wrap the character(s) in curly braces, e.g.,
> 
> title = "Pascal, {C}, {Java}: were they all conceived in {ETH}?"
> ```
> 
> So I think you want to convert Monte Carlo to {Monte} {Carlo} in the .bib file
> 
> If you have run out of time and want me to take a crack at it, I will try

This is exactly what I was thinking of, sorry I did not mention it. Please protect uppercase with curly braces.

**Comment by sjswerdloff** [comment]
Nope, haven't started...  happy to re-review when you're done

**Comment by Matthew-Jennings** [comment]
@sjswerdloff & @osorensen, [curly braces worked](https://github.com/pymedphys/pymedphys/suites/8302635439/artifacts/364291819) - thank you both!

**Comment by Matthew-Jennings** [comment]
> brackets missing on Varian at line 58 brackets missing on Monte Carlo-based at line 133 bracket missing on Yttrium-90 at line 184 brackets missing 2300C/D at line 234

Apologies, I should've looked more carefully

File: paper.bib [hunk 1]
```diff
@@ -130,7 +130,7 @@ @article{lysakovski2021development
 
 @article{castle2022,
   author={Castle, James R and Duan, Jingwei and Feng, Xue and Chen, Quan},
-  title={Development of a virtual source model for Monte Carlo-based independent dose calculation for varian linac},
+  title={Development of a virtual source model for {Monte} {Carlo}-based independent dose calculation for varian linac},
```
Comment (sjswerdloff) [inline paper.bib:133]: At first I thought that this lower case varian was in need of capitalisation and curly braces... but that is actually how it reads in the publication!

**Review by sjswerdloff — APPROVED** [review]

File: paper.bib (hunk 1 above)
Comment (Matthew-Jennings) [reply paper.bib:133]: Haha, well there you go!

**Pull request merged**